docs/dataframe_api.ipynb

```diff
@@ -0,0 +1,676 @@
+{
```

Comment:
<p>I'd suggest moving the first comment to be a text cell (it is explaining the next part of the notebook).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/tensorboard/pull/3744/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
Done.